tests/ParserTest.elm

```diff
@@ -543,6 +543,22 @@ expr =
                   , "'A'"
                   , Ok (Literal (Char 'A'))
                   )
+                , ( "double quote"
+                  , "'\"'"
+                  , Ok (Literal (Char '\"'))
```

Comment:
the double-quote char on the `Ok ...` line doesn't need to be escaped, but that's just a nitpick :slightly_smiling_face: